Gopkg.toml

```diff
@@ -0,0 +1,15 @@
+# Refer to https://golang.github.io/dep/docs/Gopkg.toml.html
+# for detailed Gopkg.toml documentation.
+
+
+[[constraint]]
+  name = "github.com/stretchr/testify"
+  version = "1.2.1"
```

Comment:
why not `^1` ? Pinning to exact version might create conflicts when opentracing is used in other projects.

Also, I am not sure if `dep` supports this, but is there a way to specify the scope of the dependency, test vs. runtime?

Reply:
> Why not `^1`? Pinning to exact version might create conflicts when opentracing is used in other projects.

The exact version was added as a result of running `dep init`. 

I don't think these files are obeyed when this library is used in other projects (although the authors of `dep` hinted that the `Glide.lock` may be used as a hint for conflict resolution in the future). 

As I understand it, the use of `dep` in libraries is more as a manifest for dependencies, and a convenient way of getting up to speed when contributing (i.e. `dep ensure` vs. `go get` * all dependencies).

That said, it's probably fine to specify `^1.2.1` so we catch all future releases in the major version. 

> Also, I am not sure if `dep` supports this, but is there a way to specify the scope of the dependency, test vs. runtime?

`dep` doesn't have the concept of test vs. runtime dependencies, although there is the concept of tooling dependencies (i.e. libraries you don't want in your `/vendor` directory) vs other dependencies (`[required]` vs `[constraint]`).

More on how `Gopkg.toml` works: https://golang.github.io/dep/docs/Gopkg.toml.html